instrumentation/instaredigo/instaredigo_test.go

```diff
@@ -0,0 +1,419 @@
+// (c) Copyright IBM Corp. 2022
```

Comment:
@sanojsubran, what do you think: does it makes sense to add tests for other constructors like `Dial...`?

Reply:
Agree. Added test cases for error condition.